### DISCUSSION TITLE
How to change punchcard to use 24h time?

### DISCUSSION THREAD

**Discussion by BrainStone** [plan-player-analytics/Plan#2391] [Q&A] [answered]
I'd love to have 24h time on the punchcard as an option, as the am/pm thing is mostly specific to the English language and most other languages use 24h time.

**Comment by AuroraLS3** [comment]
You can change the formatting method in punchcard.js for changing this before an update.

**Comment by BrainStone** [comment]
Is there a way to do that without modifying the js in the jar?  
Sorry if there’s documentation on that, I can’t really look for it because I’m on my phone.

**Comment by AuroraLS3** [comment]
https://github.com/plan-player-analytics/Plan/wiki/Html-Customization

There is a link to the wiki in the README on Repo root

**Comment by BrainStone** [comment]
Perfect. Thanks.

**Comment by AuroraLS3** [comment, marked as answer]
Following change uses 24 hour clock

```
            dateTimeLabelFormats: {
                hour: '%H:00',
                day: '%H:00'
            },
```
See http://www.php.net/manual/en/function.strftime.php for format reference

![1](http://puu.sh/BmTOo/dee12bd016.png)
